x/rollapp/types/genesis_test.go

```diff
@@ -167,10 +152,7 @@ func TestGenesisState_Validate(t *testing.T) {
 		{
 			desc: "invalid DeployerWhitelist",
```

Comment:
https://www.youtube.com/watch?v=YP3W-E0OamU

Reply:
what are you referring to?

Reply:
about the whitelist. doesn't matter if you're black or white... nvm

Reply:
sure sure
but no problem I take it 😄